Chapter_5/glue/README.md

```diff
@@ -1,6 +1,6 @@
 ## Creating a Glue Job for ETL
 
-TODO: introduction paragraph
+We will show with step-by-step on how to create a glue job for ETL. 
```

Comment:
In this file, we describe how to create a Glue job for ETL step by step.

Reply:
done